### ISSUE TITLE
"msg": "No package matching 'postgresql-9.5-postgis-2.2-scripts' is available"

### ISSUE THREAD

**Issue by MounirMesselmeni** [ANXS/postgresql#156]
Defining:
`postgresql_version: 9.5` and `postgresql_ext_postgis_version: "2.2"` raise No package matching 'postgresql-9.5-postgis-2.2-scripts' is available in Ubuntu

**Comment by easyecho** [comment]
@MounirMesselmeni Can you please provide more information -- like the release of ubuntu?  Are you setting a custom postgresql_apt_repository?  Have you done an apt-cache search for the package?

**Comment by MounirMesselmeni** [comment]
`/etc/apt/sources.list.d/apt_postgresql_org_pub_repos_apt.list` contains the following:
```
deb http://apt.postgresql.org/pub/repos/apt/ trusty-pgdg main
deb http://apt.postgresql.org/pub/repos/apt/ trusty-pgdg main 9.5
```
And for Ubuntu release it's the following:
Distributor ID:	Ubuntu
Description:	Ubuntu 14.04.4 LTS
Release:	14.04
Codename:	trusty

**Comment by easyecho** [comment]
@MounirMesselmeni Well I can confirm that package isn't available for me either.  Postgresql doesn't seem to have released a postgis 2.2 scripts package.  You would need to head over there to find out why.

**Comment by easyecho** [comment]
@MounirMesselmeni actually a little bit of digging turned up this message on the 2.1 scripts package:  

> Description: PostGIS for PostgreSQL 9.4 -- scripts -- dummy package
 This package contained SQL scripts to upgrade from earlier PostGIS
 versions for PostgreSQL 9.4 as well as Perl utility scripts.
 It got renamed to postgresql-9.4-postgis-scripts.

Seems like `- "postgresql-{{postgresql_version}}-postgis-{{postgresql_ext_postgis_version}}-scripts"` should be changed to `- "postgresql-{{postgresql_version}}-postgis-scripts"` in postgresql/tasks/extensions/postgis.yml

**Comment by MounirMesselmeni** [comment]
@easyecho Make sense and thanks for your help! Should I wait for a new release, propose a patch or fix it locally?

**Comment by easyecho** [comment]
@MounirMesselmeni I think you have enough here to propose a patch.  While your waiting for that you can fork the project and make the necessary changes so you don't hold yourself up.

Full disclosure -- I'm not a project maintainer.

**Comment by MounirMesselmeni** [comment]
@easyecho I have more than enough, thanks again for your help :+1:

**Comment by jalessio** [comment]
@easyecho I took your suggested fix and put it into a PR @ https://github.com/ANXS/postgresql/pull/160

**Comment by lhoss** [comment]
:+1:  to get this issue fixed!
There's another PR for this issue https://github.com/ANXS/postgresql/pull/164 (14d old)
with a bit misleading name (as it's not really a typo , but an outdated package name)
that interestingly shows the travis tests failed, only for ansible 1.9.x for a subset of postgres versions
(seems like a timeout, possibly apt-get waiting for user input?), see:
https://travis-ci.org/ANXS/postgresql/builds/116647257

**Comment by gclough** [comment]
Fixed in #310.  Can you verify and close this @MounirMesselmeni  issue?
